### PR TITLE
removed ra dependencies and fixed epid retrieval

### PR DESCRIPTION
**Pull request by bvavala**
Signed-off-by: Bruno Vavala <bruno.vavala@intel.com>

### CODE REVIEW

File: eservice/pdo/eservice/enclave/enclave/base.cpp [hunk 1]
```diff
@@ -140,8 +140,8 @@ pdo_err_t pdo::enclave_api::base::GetEpidGroup(
 
      try {
         // Get the EPID group from the enclave and convert it to big endian
-        sgx_epid_group_id_t epidGroup;
-        g_Enclave.GetEpidGroup(epidGroup);
+        sgx_epid_group_id_t epidGroup = { 0 };
+        g_Enclave.GetEpidGroup(&epidGroup);
```
Comment (cmickeyb) [inline eservice/pdo/eservice/enclave/enclave/base.cpp]: so... does this require the struct or a pointer? the copy below treats this as a pointer. could you verify the appropriate use of this structure.

File: eservice/pdo/eservice/enclave/enclave/enclave.cpp [hunk 2]
```diff
@@ -119,34 +126,24 @@ namespace pdo {
...
             memcpy_s(
                 outEpidGroup,
                 sizeof(sgx_epid_group_id_t),
-                remoteAttestationMessage1.gid,
+                &this->epidGroupId,
                 sizeof(sgx_epid_group_id_t));
         } // Enclave::GetEpidGroup
-
+        
         // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (cmickeyb) [inline eservice/pdo/eservice/enclave/enclave/enclave.cpp]: whitespace. please remove

File: eservice/pdo/eservice/enclave/enclave/base.cpp (hunk 1 above)
Comment (bvavala) [reply eservice/pdo/eservice/enclave/enclave/base.cpp]: I've fixed the inconsistencies. The structure is declare, but after that we use pointer to that structure.

File: eservice/pdo/eservice/enclave/enclave/enclave.cpp (hunk 2 above)
Comment (bvavala) [reply eservice/pdo/eservice/enclave/enclave/enclave.cpp]: done

**Pull request merged**